src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/RedisBatch.cs

```diff
@@ -55,36 +68,48 @@ public static object ExecuteAsync<T>(object redisBase, object message, object pr
         /// <param name="message">The message</param>
         /// <param name="processor">The result processor</param>
         /// <param name="server">The server</param>
+        /// <param name="callOpCode">The <see cref="OpCodeValue"/> used in the original method call.</param>
         /// <returns>An asynchronous task.</returns>
-        private static async Task<T> ExecuteAsyncInternal<T>(object redisBase, object message, object processor, object server)
+        private static async Task<T> ExecuteAsyncInternal<T>(object redisBase, object message, object processor, object server, OpCodeValue callOpCode)
         {
             var thisType = redisBase.GetType();
+
+            if (_redisAssembly == null)
+            {
+                // get these only once and cache them,
+                // no need for locking, race conditions are not a problem
+                _redisAssembly = thisType.Assembly;
```

Comment:
Nice! I feel like I should edit my GraphQL integration to do this since the GraphQL parameters are defined in the same assembly.

Reply:
I'm ok with doing it either way (like this or using the assembly name). In this case the assembly name can be either `StackExchange.Redis` or `StackExchange.Redis.StrongName`, so doing it this way allows us to handle either one.